src/universal/conf/app.conf

```diff
@@ -0,0 +1,23 @@
+# This is the base configuration file for the Mantis ETC client.
+
+# This where all the default settings are defined (this file is packaged within the executable).
+# It should always go at the top.
+include "application.conf"
+
+# Enable HTTPS
```

Comment:
I prefer to move the https config to a separate file and here add only `include https.conf`

Reply:
As commented [here](https://github.com/input-output-hk/mantis/pull/681#discussion_r492147771) I will remove for now this configuration